src/admin.py

```diff
@@ -166,11 +168,15 @@ def debug_uptime(_id, endpoint=None):
 def resave():
     # when index mappings are changed
     logger = logging.getLogger("resave")
-    for smartapi in SmartAPI.get_all(1000):
+    for smartapi in SmartAPIEntity.get_all(1000):
         logger.info(smartapi._id)
         smartapi.save()
 
 
+def refresh_metakg(include_reasoner=False):
```

Comment:
@sengineer0 let's set `include_reasoner` as True by default.

Reply:
Sure, will update soon.

Reply:
it's updated @newgene